www/rules/index.html

```diff
@@ -105,7 +105,20 @@ <h4>7. Naming your map</h4>
     </li>
   </ul>
 
-  <h2>🗃️ Server Type Rules</h2>
+  <h2>🗃️ Hosting & Server Type Rules</h2>
+
+  <h4>🌐 Hosting rules</h4>
+  <ul>
+    <p>Hosting publicly accessible servers requires them to be registered with Teeworlds' and DDNet master servers.
+      Game servers that register with these master servers are subject to <a href="https://www.teeworlds.com/?page=docs&wiki=rules/server_rules">the server hosting rules of Teeworlds</a> as well as the additional DDNet rules:</p>
+    <li>A modified Teeworlds server must not use a standard gametype (dm, tdm, ctf, lms, lts)</li>
+    <li>0.6.x and previous: The player count shown must resemble the amount of human players. The maximum player count must be the maximum number of human players.</li>
+    <li>0.7.x and later: You can also include non-human players using a special bot flag. This allows client-side filtering.</li>
+    <li>Do not host an excessive amount of servers. There is no fixed limit but try to not overdo it. (i.e. there should be people playing on them eventually)</li>
+    <li>Do not host servers purely for advertising purposes (e.g. passworded & having only an advertisement as name; having sponsoring info or the likes in the server name is perfectly fine).</li>
```

Comment:
```suggestion
    <li>Do not host servers purely for advertising purposes (e.g. a passworded server only for advertising purposes); having sponsoring info or the likes in the server name is perfectly fine).</li>
```

Reply:
that's copy and pasted from teeworlds

Reply:
fair - i just dislike how it reads :P

Reply:
I know what you mean, gotta get more feedback. if more ppl are for changing that, we can do that.